packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -427,6 +432,36 @@ export class StripeWebhookHandler extends StripeHandler {
     await this.updateCustomer(uid, email);
   }
 
+  /**
+   * Validate product metadata on update
+   */
+  async handleProductUpdate(request: AuthRequest, event: Stripe.Event) {
+    const product = event.data.object as Stripe.Source;
```

Comment:
I don't understand why this is a Stripe.Source.  Shouldn't it be a Stripe.Product?